aos_cd_jobs/common.py

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/env python
 from os import getenv
-from os.path import join
+from os.path import exists, join
 
 from git import Repo
 
 JOBS_DIRECTORY = 'jobs'
 
 def initialize_repo():
     repo_dir = join(getenv('WORKSPACE'), 'aos-cd-jobs')
+    if exists(repo_dir):
+        return Repo(repo_dir)
```

Comment:
I think we want to clear out the working directory on the job and start from scratch here so we have fewer headaches about what local git state does and does not exist. Thoughts?